crates/executor/src/revm_wrap.rs

```diff
@@ -192,6 +192,15 @@ pub fn is_log_equal(revm_log: &revm::Log, reth_log: &reth_primitives::Log) -> bo
             .any(|(revm_topic, reth_topic)| revm_topic.0 != reth_topic.0)
 }
 
+/// Into reth primitive [Log] from [revm::Log].
+pub fn into_reth_log(log: revm::Log) -> Log {
+    Log {
+        address: H160(log.address.0),
+        topics: log.topics.into_iter().map(|h| H256(h.0)).collect(),
+        data: log.data.into(),
+    }
+}
+
```

Comment:
Should be able to impl `revm_primitives::Log for `reth_primitives::Log` (even better if we could re-use the type)

Reply:
I've already discussed it w/ @onbjerg. Please note https://github.com/paradigmxyz/reth/pull/1096#discussion_r1093254550